src/execution/__tests__/stream-test.ts

```diff
@@ -450,6 +450,56 @@ describe('Execute: stream directive', () => {
       },
```

Comment:
Please ensure that all the resources, like database connections, are being handled appropriately.